clippy_lints/src/format.rs

```diff
@@ -69,11 +69,10 @@ impl LateLintPass for Pass {
     }
 }
 
-/// Checks if the expressions matches
-/// ```rust
-/// { static __STATIC_FMTSTR: &[""] = _; __STATIC_FMTSTR }
-/// ```
-fn check_static_str(cx: &LateContext, expr: &Expr) -> bool {
+/// Returns the slice of format string parts in an `Arguments::new_v1` call.
+/// Public because it's shared with a lint in print.rs.
```

Comment:
Small nit: I had made a `higher` module for that kind of un-lowering/un-expanding stuffs.